util/testkit/testkit.go

```diff
@@ -326,3 +326,19 @@ func (tk *TestKit) GetTableID(tableName string) int64 {
 	tk.c.Assert(err, check.IsNil)
 	return tbl.Meta().ID
 }
+
+// AssertErrorAndRetry asserts the error and retries the sql.
+func (tk *TestKit) AssertErrorAndRetry(errCode, maxRetries int, sql string, args ...interface{}) {
+	for i := 0; i <= maxRetries; i++ {
+		tk.c.Assert(i < maxRetries, check.IsTrue, check.Commentf("sql: %s, args: %v, retries: %d", sql, args, maxRetries))
+		res, err := tk.Exec(sql, args...)
+		if err != nil {
+			tk.MustGetErrCode(err.Error(), errCode)
```

Comment:
The first argument of `MustGetErrCode` is a SQL.

Reply:
@djshow832 Please address the comment.